use-cases/guest-technology.md

```diff
@@ -0,0 +1,21 @@
+## Guest technology
+
+Guest Technology integrations such as telephone systems are used for staff to identify guests on telephones or TVs.
+
+### Setup
+
+The integration should use [Get all services](../operations/services.md#get-all-services) operation to retrieve all services the property has configured in Mews. Once all services are retrieved, select the service to which you want to attribute the charges. Alternatively, a field where the `ServiceId` can be entered can be used.
+
+The [Get all accounting categories](../operations/finance.md#get-all-accounting-categories) operation should be used to retrieve all accounting categories has configured in Mews. This is important as a hotel may prefer to have charges for international phone calls reported with a different accounting category than domestic phone calls.
+
+### Room Status
+
+Guest Technology integrations are required to receive an update to a Reservation State in real time, this is where the [Reservation Websocket](../websockets#reservation-event) should be used to avoid constantly polling for new commands.
```

Comment:
```suggestion
Guest Technology integrations are required to receive an update to a Reservation State in real time, this is where the [Reservation Websocket](../websockets.md#reservation-event) should be used to avoid constantly polling for new commands.
```